Contributing.md

```diff
@@ -0,0 +1,49 @@
+## Contributor Guidelines
+
+We encourage all members of the Aberdeen Centre for Health Data Science to contribute to this team project :smile:
+
+Here is what you need to do:
+
+**1. Set up a Github account**
+
+Please set up a free [GitHub](https://github.com/) account and sign in. Here are some [instructions](https://help.github.com/articles/signing-up-for-a-new-github-account). Let [Dimitra](https://github.com/dblana) know you've done this so we can add you to the ACHDS organisation.
+
+**2. Decide on how you'd like to contribute**
+
+There are many different ways to contribute to this research project:
+
+[TO DO: discuss the main aspects of the research project and come up with a list of issues describing the main tasks]
+
+You might also just like to suggest improvements, but not make the changes yourself. This is also very welcome! In this case, you just need to do step 3 below: add an issue describing your suggested improvement.
+
+**3. Add an issue describing your suggested contribution (or assign an existing issue to yourself)**
+
+Once you've identified a way to contribute, please let us know what you're planning to do by adding a new issue [here](https://github.com/AbdnCHDS/covid-social-distancing/issues), or assign an existing issue to yourself. Issues are basically tasks that we want to achieve, so by adding your issue to the list we can keep track of who is working on what.
+
+Please don't skip this step, as knowing what you're planning on doing in advance ensures you aren't overlapping with work that's currently underway and that everyone is on the same page with the goal of the work you're going to carry out. Here is [more information about Github issues](https://docs.github.com/en/github/managing-your-work-on-github/managing-your-work-with-issues).
+
+**4. Create a new branch (and give it a name associated with the issue)**
+
+Using a separate [branch](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-branches) allows you to isolate development work without affecting other branches in the repository.
+
+**5. Create a draft pull request and add "closes #[issue number]" in the description**
+
+A [pull request](https://help.github.com/en/articles/proposing-changes-to-your-work-with-pull-requests) is the method by which you let us know of the contribution you have made. Adding "closes #[issue number]" in the description [links the pull request to the issue](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword). This shows that work is in progress to address the issue, and it automatically closes the issue when the pull request is merged.
+
+**6. Go ahead and make your changes in the new branch**
+
+Make sure you commit your changes frequently, and use good commit messages that briefly describe the changes you made.
+
```

Comment:
Testing a comment on a line. Looks good.

Reply:
I love that we can be very specific about what a comment refers to 😃